### PR TITLE
Unit Tests for Merfolk Trickster, confirming #5825

### PR DESCRIPTION
**Pull request by drmDev**
Added a small batch of unit tests for Merfolk Trickster interactions, and seems to confirm #5825  - the Devil token kills Trickster. Footlight Fiend does not however, so I suspect this is an issue related to Tokens. Maybe I'll add more tests with Trickster interacting with other tokens to confirm those work correctly?

### CODE REVIEW

**Comment by drmDev** [comment]
Since a Unit Test will fail here due to possible bug in code (assuming my tests are valid), the Travis CI build will likely fail from it.

File: Mage.Tests/src/test/java/org/mage/test/cards/continuous/MerfolkTricksterTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        
+        assertLife(playerA, 20);
+        assertLife(playerB, 20);
+        assertTappedCount("Island", true, 2);
+        assertPermanentCount(playerB, mTrickster, 1);
+        assertDamageReceived(playerB, mTrickster, 1);
+        //assertAllCommandsUsed(); // uncommenting this will force a failure since PlayerA cannot do a command to target Trickster, as expected
+    }
+    
+    // Making sure Footlight Fiend working correctly here first...
```
Comment (t-my) [inline Mage.Tests/src/test/java/org/mage/test/cards/continuous/MerfolkTricksterTest.java]: Perhaps this test case should be in Footlight Fiends file.

Comment (drmDev) [reply Mage.Tests/src/test/java/org/mage/test/cards/continuous/MerfolkTricksterTest.java]: Fair point. I was just lazy initially! I'll move it outside of here.

**Comment by drmDev** [comment]
Moved the FootlightFiend test outside as requested.

**Pull request merged**